docker/android-slave-s2i/tools/android-wait-for-emulator.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
We do not need this anymore. Correct me if I'm wrong? From where this is coming from?